web/e2e-tests/realm-playground.test.ts

```diff
@@ -35,7 +35,7 @@ async function test_successful_playground_creation(page: Page): Promise<void> {
     const payload = {
         pygments_language: "Python",
         playground_name: "Python3 playground",
-        url_prefix: "https://python.example.com",
+        url_template: "https://python.example.com{?code}",
```

Comment:
Probably worth changing these to match our new documentation as well.

Reply:
Yeah. Changed the examples to `?code={code}`. Also added an excluded pattern to `custom_check.py` for `"pattern": "={"`.